modules/weboramaRtdProvider.js

```diff
@@ -682,16 +673,6 @@ class WeboramaRtdProvider {
   // eslint-disable-next-line no-dupe-class-members
   #handleBid(reqBidsConfigObj, bid, profile, metadata) {
     this.#handleBidViaORTB2(reqBidsConfigObj, bid.bidder, profile, metadata);
-
-    /** @type {Object.<string,string>} */
-    const bidderAliasRegistry = adapterManager.aliasRegistry || {};
-
-    /** @type {string} */
-    const bidder = bidderAliasRegistry[bid.bidder] || bid.bidder;
-
-    if (bidder == 'appnexus') {
-      this.#handleAppnexusBid(reqBidsConfigObj, bid, profile);
```

Comment:
Don't have a good place to insert this comment; but from testing weborama they set their data in the `site.ext.data` instead of the ortb segment fields, so it's not getting included/read by the new logic for the appnexus bid adapter.

![Screen Shot 2023-06-13 at 8 32 40 AM](https://github.com/prebid/Prebid.js/assets/31102355/de4075ce-3da4-451f-83f4-8285dcce51d5)

Reply:
Correct, this is intentional and called out in the OP. If we end up keeping the key=value parsing I'll also undo this.